src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo"></a>
+      <nav class="nav">
+        <ul class="ul__list">
+          <li class="nav__list">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__list">
```

Comment:
this is not a list, it is a list item. the appropriate class name would be `nav__item`